forte/data/extractor/attribute_extractor.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Dict, Any, Union, Iterable
+from ft.onto.base_ontology import Annotation
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.converter.feature import Feature
+from forte.data.extractor.base_extractor import BaseExtractor
+
+
+class AttributeExtractor(BaseExtractor):
+    '''This type of extractor will get the attribute on entry_type
```

Comment:
Do we need to add a `r` here? Like this:
```
r"""This type of extractor will get the attribute on entry_type
"""
```

Reply:
'r' stands for raw string, so it is useful when we have some special symbols in the docstring. For this case we should be fine.

Reply:
Got it.